test/cli/sbom_input_test.go

```diff
@@ -0,0 +1,130 @@
+package cli
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+// GRYPE_BINARY_LOCATION is relative to the repository root. (e.g., "snapshot/grype_linux_amd64/grype")
+// This value is transformed due to the CLI tests' need for a path relative to the test directory.
+var grypeBinaryLocation = path.Join("..", "..", os.Getenv("GRYPE_BINARY_LOCATION"))
+
+const sbomLocation = "./test-fixtures/sbom-ubuntu-20.04--pruned.json"
+
+func TestSBOMInput_AsArgument(t *testing.T) {
+	workingDirectory, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cases := []struct {
+		name string
+		path string
+	}{
+		{
+			"absolute path",
```

Comment:
I wasn't aware that you could define structs without their field names. When is this style preferred?

Reply:
I don't think I have a good answer to that question offhand. In this case, I had two reasons for this syntax, 1) the field names appeared immediately before the data (in the struct type literal), and 2) in my editor (GoLand), the field names actually do appear as hints next to the field values.

Having said all that, I don't mind changing to include field names 😃